dashboard/app/mailers/pd/application/teacher_application_mailer.rb

```diff
@@ -1,5 +1,6 @@
 module Pd::Application
-  class TeacherApplicationMailer < ActionMailer::Base
+  class TeacherApplicationMailer < ApplicationMailer
+    include ActionMailerMetrics
```

Comment:
You don't need this since it's already included in the parent class `ApplicationMailer`